Specifications/FeelbotTalking3.feature

```diff
@@ -0,0 +1,6 @@
+Scenario : FeelBotTalking3
+This scenario is about the conversations we are able to have with FeelBot
+
+Given : That FeelBot is able to analyze words that are said to her
+When : I talk to FeelBot on my laptop or my phone
+Then : FeelBot is able to respond to my questions
```

Comment:
Isn't it a bit too vague ?

What kind of questions ? What kind of answers ? 
Does googling it suffice ?